lib/App.js

```diff
@@ -2171,13 +2271,25 @@ class App {
 
   }
 
-  static async addTypes({ appPath }) {
+  static async addTypes({ appPath }, usesTypeScript) {
     // Check if npm is available, then install homey type declarations as dev dependency
     const npmInstalled = await NpmCommands.isNpmInstalled();
     if (npmInstalled) {
       await NpmCommands.install({ saveDev: true, path: appPath }, {
         id: '@types/homey@npm:homey-apps-sdk-v3-types'
       });
+
+      if (usesTypeScript) {
```

Comment:
If you make `usesTypeScript` static we can just call it here and it works with `homey app add-types` too.
Alternatively we make `addTypes` non-static and we can call `usesTypeScript` here as is (if you do this make sure to update `/bin/cmds/app/add-types.js` so it creates a `new App()`.

Reply:
Good suggestion :)